src/elements/OcButton.vue

```diff
@@ -89,9 +96,16 @@ export default {
       },
     },
     /**
-     * Set the button’s icon to display.
+     * Set the button’s icon to be display on the left side.
      */
-    icon: {
+    iconLeft: {
```

Comment:
I would add a News prop which defines the Position of the icon. Default is left as before.
This requires less changes in Phoenix

Reply:
That wouldn’t cover the situation when icon should be on both sides and both of them should be different.

Reply:
I always forget to put screenshots here in ODS 🤦‍♂ I added one now to showcase it

Reply:
I see - let me think about this until Monday :wave:

Reply:
Some example code from vuetify:

```
 <v-btn
      :loading="loading3"
      :disabled="loading3"
      color="blue-grey"
      class="white--text"
      @click="loader = 'loading3'"
    >
      Upload
      <v-icon right dark>cloud_upload</v-icon>
    </v-btn>

Reply:
the icon is not a property of the button but in the inner element.

I'd consider this a more reasonable approach - there might be more things we need to add to the button one day ....

Reply:
Not really sure about this. I'd say it would be better as a prop and not having to always really write the icon inside (not really much difference I know but still :grin:) And I'd say adding more stuff would make the button just more complicated so I wouldn't recommend to go that direction.

Reply:
I like that example https://github.com/owncloud/owncloud-design-system/pull/418#discussion_r303298587 the most as it's the most flexible.

Encasing more and more and adding more pros bloats the code and makes things rather stiff